src/game-logic/reducers/waterField.js

```diff
@@ -0,0 +1,10 @@
+import { setWasWatered, updateField } from './helpers'
+
+/**
+ * @param {farmhand.state} state
+ * @returns {farmhand.state}
+ */
+export const waterField = state => ({
```

Comment:
tests 🏴

Reply:
Added a TODO for this in https://github.com/jeremyckahn/farmhand/commit/b28490644c45a589008d909fc85add26f51d46df.